### PR TITLE
formatted with yapf

### PR DESCRIPTION
**Pull request by pacogomez**
To make pep8 validation easier, I ran `yapf` against `pyvcloud/vcd/*.py`. This PR only has formatting changes, it *should* not have any changes in the logic.

### CODE REVIEW

File: pyvcloud/vcd/utils.py [hunk 1]
```diff
@@ -170,12 +167,16 @@ def vapp_to_dict(vapp, metadata=None):
                         value = '{:,} {}'.format(int(quantity), units).strip()
                 else:
                     value = '{}: {}'.format(
-                                        connection.get('{http://www.vmware.com/vcloud/v1.5}ipAddressingMode'),  # NOQA
-                                        connection.get('{http://www.vmware.com/vcloud/v1.5}ipAddress'))  # NOQA
+                        connection.get(
+                            '{http://www.vmware.com/vcloud/v1.5}ipAddressingMode'  # NOQA
```
Comment (rocknes) [inline pyvcloud/vcd/utils.py]: We should replace all these hard coded strings with the corresponding item in NSMAP in a future commit.

File: pyvcloud/vcd/vdc.py [hunk 2]
```diff
@@ -201,42 +192,56 @@ def instantiate_vapp(self,
...
                         item['{' + NSMAP['rasd'] + '}VirtualQuantity'] = memory
                         memory_params = item
                         virtual_hardware_section.append(memory_params)
 
                 if cpu is not None and cpu_params is None:
                     if item['{' + NSMAP['rasd'] + '}ResourceType'] == 3:
-                        item['{' + NSMAP['rasd'] + '}ElementName'] = '%s virtual CPU(s)' % cpu  # NOQA
+                        item[
+                            '{' + NSMAP['rasd'] +
+                            '}ElementName'] = '%s virtual CPU(s)' % cpu  # NOQA
```
Comment (rocknes) [inline pyvcloud/vcd/vdc.py]: Should we remove the no quality assurance tags from the formatted lines?

File: pyvcloud/vcd/utils.py (hunk 1 above)
Comment (pacogomez) [reply pyvcloud/vcd/utils.py]: will remove it in my current development branch

File: pyvcloud/vcd/vdc.py (hunk 2 above)
Comment (pacogomez) [reply pyvcloud/vcd/vdc.py]: done

**Pull request merged**